.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  linux:
+    name: "Node ${{ matrix.node }} on Linux: Lint"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - 10
+          - 12
+          - 14
+          - 16
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
I wonder, should we get all the commits here? I mean, say you push a patch with the proper commit message, this would still flag older commits, but you can fix this with squash when merging.

Reply:
I have proposed that we use rebase merging always. See https://github.com/karma-runner/karma/issues/3773.

Reply:
Rebase is good too, but if the patches all refer to one change, squash makes more sense IMHO.

Reply:
[c825542](https://github.com/karma-runner/karma-jasmine/pull/313/commits/c8255429284effe5f74d9538c024ae87054d6cc4) done.

Reply:
In that case, squashing should be done in the head branch, prior to merging. Does that sg to you?

Reply:
@XhmikosR

Reply:
Any other thoughts before merging? @XhmikosR

Reply:
Sorry, I don't have a lot of time nowadays :/ I'd say just proceed with it and drop Node.js 10 later. I still think we shouldn't use the minimum Node.js version here because it's a dev task, but I'll let you decide about it.